docs/dictionary/function/revOpenDatabase.lcdoc

```diff
@@ -3,15 +3,15 @@ Name: revOpenDatabase
 Synonyms: revdb_connect
 
 Type: function
-Syntax: revOpenDatabase(<databaseType>, {<hostAddress>, <databaseName>, [<userName>],[<passWord>],{[<useSSL>], [<socketAddr>], [<rwTimeout>], [<autoReconnect>] | [<cursorType>] | [<sslOption>],...} | <filePath>,[<sqliteOptions>]})
+Syntax: revOpenDatabase(<databaseType>, {<hostAddress>, <databaseName>, [<userName>],[<passWord>], {[<useSSL>], [<socketAddr>], [<rwTimeout>], [<autoReconnect>] | [<cursorType>] | [<sslOption> ...]} | <filePath>,[<sqliteOptions>]})
```

Comment:
@livecodeali looking at this I'm thinking that the other syntax definitions below actually fully cover this one? Perhaps this one can be removed given how differently the function behaves depending on the database type I think it makes sense to list them separately.

Reply:
Agreed